packages/rancher-monitoring/prometheus-federator/generated-changes/patch/Chart.yaml.patch

```diff
@@ -12,4 +12,8 @@
 +  catalog.cattle.io/rancher-version: '>= 2.10.0-0 < 2.11.0-0'
    catalog.cattle.io/release-name: prometheus-federator
  apiVersion: v2
- appVersion: 0.4.3
+-appVersion: v0.4.4-rc.1
++appVersion: v0.4.4-rc.2
```

Comment:
I'm confused by the versioning here. I thought we'd decided to not bump rc numbers for monitoring charts? Plus we have the `release.yaml` showing `105.1.0-rc.1+up0.4.5-rc.1` but all these are showing `-rc.2`

Reply:
`appVersion` in helm should represent the version of the image/binary, where as `version` is about the chart itself. These don't need to match for any technical reason. Note this is also why it's chart version `0.4.5` (with an RC) and image tag `0.4.4` (with an RC) as part of the difference along with the RC number being different.

I don't believe we have a policy based requirement that App and Chart versions match 1:1. And if we do then it creates a chicken-egg problem in the current `prometheus-federator` repo - as charts values would need reference versions of images that don't exist.

As for comparing the chart version, `rancher/charts` appends the "upstream chart version" onto the end. So in cases where the image version (`appVersion`) and chart version do not match it's expected to see a difference there.

I was under the impression the "no bump RC for monitoring" was just for the monitoring charts from upstream. I thought that since PromFed is something we produce completely that we do full RC process?

Reply:
Sounds good to me, I was confused / just wanted to confirm since the versions were so close to each other